src/pages/FriendShipPage/FriendShipPage.tsx

```diff
@@ -53,11 +53,12 @@ export const FriendShipPage: React.FunctionComponent = () => {
             }`}
           >
             <TablesUserList
-              venueName={venue.name}
+              venue={venue}
               setSeatedAtTable={setSeatedAtTable}
               seatedAtTable={seatedAtTable}
               TableComponent={TableComponent}
               customTables={FRIENDSHIP_CUSTOM_TABLES}
+              defaultTables={FRIENDSHIP_CUSTOM_TABLES}
```

Comment:
is this FriendshipPage used anywhere? it feels like it's a legacy one.

Reply:
Nope, it's legacy